### PR TITLE
added missing return statement in scan_sources() method

### PR DESCRIPTION
**Pull request by amrshindhe**
there was a missing return statement to return the sources from scan_sources method

### CODE REVIEW

File: dbcat/api.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def scan_sources(
                 scanner.scan()
             except StopIteration:
                 raise NoMatchesError
-
```
Comment (vrajat) [inline dbcat/api.py:143]: Can you explain how you use this function in your code?  The original intent of this function is to only scan and not return the list of sources.

Comment (amrshindhe) [reply dbcat/api.py:143]: @vrajat sir, we are scanning resources, that will return list of sources specified by parameters in scan_sources(), we are using those sources to scan_database, below is the code snippet for that
Screenshot 2023-11-30 at 5 12 54 PM

Comment (vrajat) [reply dbcat/api.py:143]: You don't need to call `scan_sources`. `scan_database` calls `scan_sources` internally. 

Ref: https://github.com/tokern/piicatcher/blob/master/piicatcher/api.py#L84

Comment (vrajat) [reply dbcat/api.py:143]: You need to get `CatSource` with `source = catalog.get_source(source_name)` and pass that into `scan_database`.

Comment (amrshindhe) [reply dbcat/api.py:143]: Oh, thank you sir, actually i couldn't find the documentation for  piicatcher api usage, if you are planning for detailed documentation i can help.

Comment (vrajat) [reply dbcat/api.py:143]: Agree - it requires more documentation. One suggestion is to add docstrings to the functions and then open PRs. Docs can be generated from that.

**Comment by vrajat** [comment]
Closing this as the PR is not required. Refer conversation.

**Pull request closed without merging**